src/site/tests/lovell/switch-links.cypress.spec.js

```diff
@@ -0,0 +1,56 @@
+const lovellSwitchLinkPages = [
+  {
+    title: 'System Health Care Page',
+    path: '',
+  },
+  {
+    title: 'Locations',
+    path: '/locations',
```

Comment:
Would these run faster if they used the direct location rather than having to wait on the redirect from /path to /path/  ?
```suggestion
    path: '/locations/',
```

Reply:
Nevermind.  I see where you are setting the trailing slash down on line 39.